README.md

```diff
@@ -98,10 +98,15 @@ Please report bugs and feature requests in the [issues](https://github.com/mne-t
 You can run a benchmark locally with:
 
 ```console
-pytest -m benchmark mne_qt_browser
+pytest -m benchmark mne_qt_browser/tests/test_speed.py
```

Comment:
You shouldn't need to do this (I don't see much advantage)

Reply:
I agree. Thanks. 
Changed here: 15c48136960bce8e102f5d50eeaa6786cbe37456